src/components/Toggle/Toggle.stories.tsx

```diff
@@ -1,36 +1,52 @@
-import { ComponentMeta } from "@storybook/react";
-
-import { Toggle } from "./Toggle";
-import { DecorativeBox } from "../../../.storybook//DecorativeBox";
-import { Box } from "../Box";
+import { Meta, StoryObj } from "@storybook/react";
 import { Text } from "../Text";
+import { Toggle } from "./index";
 
-export default {
+const meta: Meta<typeof Toggle> = {
+  title: "Components / Toggle",
+  tags: ["autodocs"],
   component: Toggle,
-} as ComponentMeta<typeof Toggle>;
+};
+
+export default meta;
+type Story = StoryObj<typeof Toggle>;
+
+export const Primary: Story = {
+  args: {
+    children: [
+      // eslint-disable-next-line react/jsx-key
+      <Text variant="body">Option 1</Text>,
```

Comment:
question: Is it big deal to add key ?
```suggestion
      <Text key="1" variant="body">Option 1</Text>,
```

Reply:
yes, I don't want this to be present within the documentation snipped code